honeybee/face.py

```diff
@@ -384,44 +384,59 @@ def set_adjacency(self, other_face, tolerance=0):
             other_face: Another Face object to be set adjacent to this one.
             tolerance: The minimum distance between the center of two aperture
                 geometries at which they are condsidered adjacent. Default: 0.
+
+        Returns:
+            adjacent_apertures -- A list of tuples with each tuple containing 2 objects
```

Comment:
any reason we don't use the same structure as args?
```suggestion
            adjacent_apertures: A list of tuples with each tuple containing 2 objects.
```

Reply:
Yes, there's a good reason. Sphinx isn't able to render the returns in the same structure as the Args but the double dash gives some clarity as you can see in the ladybug docs hre:
https://www.ladybug.tools/ladybug/docs/ladybug.datacollection.html#ladybug.datacollection.DailyCollection.filter_collections_by_statement

Reply:
I see. In that case I suggest to follow something like this one instead of coming up with a new syntax: https://www.sphinx-doc.org/en/1.5/ext/example_google.html